### ISSUE TITLE
Add troubleshooting to SCS Smoke Tests docs

### ISSUE THREAD

**Issue by csterwa** [pivotal-cf/docs-spring-cloud-services#114]
At https://docs.pivotal.io/spring-cloud-services/1-5/smoke-tests.html we should add some content on issues such as with:

* Azure networking - KB article -> https://discuss.pivotal.io/hc/en-us/articles/115005583008-Azure-Networking-Connection-Idle-for-more-than-Four-minutes
* Timeouts due to long app deployments

**Comment by fifthposition** [comment]
This is up at http://docs-pcf-staging.cfapps.io/spring-cloud-services/1-5/smoke-tests.html#troubleshooting.

@csterwa, please LMK if there's anything more to cover regarding the second "Troubleshooting" entry (about long application deployments).

**Comment by csterwa** [comment]
@fifthposition I like the troubleshooting section. 

Here is some feedback for potential changes:

* Where it says `To allow for longer service instance provisioning times:`, the actual issue is not provisioning times but rather that the connections get closed without notice to the service broker if the upstream networking is not configured for idle timeouts longer than the application default connection timeouts. This issue could happen to other applications, as well, as we saw with multiple customers own applications now.

* The 2nd bullet under that is probably best for the following troubleshooting section instead: `Check the setting of the Spring Cloud Services “App push timeout” tile configuration option. The default setting is 4 minutes. Change this to 15 minutes.`. As mentioned above, this first troubleshooting item is not about push timeouts but rather connections getting closed without notice on the IaaS or upstream networking appliances.

* 8 minutes as a potential setting on the push timeouts for the 2nd troubleshooting section may be alright. I would say something like `greater than the time it is taking to create a SCS service instance` or maybe rewording to give some idea that this can be increased to wait longer for provisioning to finish if the time is slower than default setting.

**Comment by fifthposition** [comment]
@csterwa Thanks! Fixed these. This is up again at http://docs-pcf-staging.cfapps.io/spring-cloud-services/1-5/smoke-tests.html#troubleshooting.
